src/directives/formly-form.test.js

```diff
@@ -952,5 +952,31 @@ describe('formly-form', () => {
       expect(listener).to.have.been.called
       expect(expression).to.have.been.called
     })
+
+    it(`should setup any watchers specified on a fieldgroup`, () => {
+      scope.model = {}
+
+      const listener = sinon.spy()
+      const expression = sinon.spy()
+
+      scope.fields = [{
+        watcher: [{
+          listener: '',
+          expression: '',
+        }, {
+          listener,
+          expression,
+        }],
+        key: 'group',
```

Comment:
ok, apparently when the expression is an empty string, this commit set it to it's field's [nested]model https://github.com/formly-js/angular-formly/commit/2b69b8e564c6d05699c9e89835fbb8e6b572c64c

I will fix it asap